### DISCUSSION TITLE
how to set portable version as default Windows console host

### DISCUSSION THREAD

**Discussion by ali50m** [microsoft/terminal#15477] [Q&A] [answered]
I am using the latest preview version [v1.18.1421.0](https://github.com/microsoft/terminal/releases/tag/v1.18.1421.0) in a Win11 laptop.

With downloading the zip file and configuring the terminal as a portable version, I donot know how to make it as the **default Windows console host**.

When using the previous stable version installing from Winget, I can easily find "Windows Terminal" option in Window settings for default console.

**Comment by DHowett** [comment, marked as answer]
Thanks for asking! This is not possible today. Please see the [distribution list](https://learn.microsoft.com/en-us/windows/terminal/distributions) for a comparison of the features available in each distribution.

**Reply by ali50m** [reply]
thanks
